tools/devcluster.yaml

```diff
@@ -70,3 +70,5 @@ stages:
         master_host: 127.0.0.1
         master_port: 8080
         container_master_host: $DOCKER_LOCALHOST
+        log:
+          level: trace
```

Comment:
Should a release note be added to communicate to end users that this config is now available?

Reply:
Added!